### ISSUE TITLE
Change Configs Plugged in and Battery

### ISSUE THREAD

**Issue by N1v0k** [Valkirie/Thunderbolt-Switch#8]
It would be very useful if different configs could be applied not only if switiching between GPUs but also between plugged in or battery.

Eg. Battery > Power Profile 15W  > 720p resolution, low Settings
Plugged In > Power Profile 35W > 1080p, medium Settings

**Comment by Valkirie** [comment]
You have that option already. In your game settings window, tick "Use power-specific settings". I'll be adding the option to have power profile specific settings soon.
![image](https://user-images.githubusercontent.com/934757/124559272-d69d4580-de3b-11eb-8d66-2d6d7bfba1f4.png)

**Comment by Valkirie** [comment]
@Gmentsik can you confirm it's working?

**Comment by N1v0k** [comment]
@Valkirie yes the power-profiles are working as expected, main reason for opening this issue was bc of the feature request for adding configuration profiles for each power profile

**Comment by Valkirie** [comment]
I'm on it. My concern is that you can have multiple power profile set as the same time so you could end up having many copies of your settings if you don't pay attention. Most likely will display a warning when that option is turned on and give the ability for people to delete settings copies.

![image](https://user-images.githubusercontent.com/934757/124950417-af4ca100-e012-11eb-9613-537eb3426754.png)
